### ISSUE TITLE
Issues while saving GeoDataframe to HDF

### ISSUE THREAD

**Issue by chmeul** [geopandas/geopandas#249]
Hi, 

I don't know whether i should post this to geopandas or pandas, but here is my issue: 

When trying to use the pandas to_hdf() function on a GeoDataframe with a "geometry" column, i get the following behavior:

1) pandas 0.16.2 and format=fixed: (warning only, **works**)
```
PerformanceWarning: 
your performance may suffer as PyTables will pickle object types that it cannot
map directly to c-types [inferred_type->mixed,key->block3_values] [items->['geometry']]
```
2) pandas 0.16.2 or 0.17.0 and format=table:
```
TypeError: Cannot serialize the column [geometry] because
its data contents are [mixed] object dtype
```
3) pandas 0.17.0 and format=fixed:
```
TypeError: objects of type ``DatetimeIndex`` are not supported in this context, sorry; supported objects are: NumPy array, record or scalar; homogeneous list or tuple, integer, float, complex or bytes
```
number 3) is more related to the way pandas 0.17.0 is dealing with datetime, but still this makes my geopandas-based code incompatible with pandas 0.17.0 or with pytables in the table format.

any hint?

**Comment by jorisvandenbossche** [comment]
@chmeul Issue 1) and 2) are expected as the geometry objects cannot be stored natively (the warning and error messages explain it). 
Issue 3 seems related to pandas in any case, and you may want to raise an issue about that over there. Are you sure you did exactly the same in pandas 0.16.2 and 0.17.0 ? (I would not expect the difference)

**Comment by jorisvandenbossche** [comment]
Ah, I think the difference between 0.16.2 and 0.17.0 will probably come from the new timezone support. Is it possible that you have a timezone aware datetime column? This is not supported with the `fixed` format (see also https://github.com/pydata/pandas/issues/11281). So if you want to keep using the `fixed` format, you need to have naive datetimes.

@jreback

**Comment by jorisvandenbossche** [comment]
@chmeul the issue you see in 3) will be fixed in pandas 0.17.1 (to be released shortly), see https://github.com/pydata/pandas/pull/11628

Closing this as further not a geopandas issue

**Comment by chmeul** [comment]
Ok, perfect, thanks!
